01-data-structures/01-introduction-to-data-structures/line/line.rb

```diff
@@ -9,26 +9,36 @@ def initialize
   end
 
   def join(person)
+    self.members.push(person)
   end
 
   def leave(person)
+    self.members.delete(person)
   end
 
   def front
+    self.members[0]
   end
 
   def middle
+    last_persons_index = self.members.length
```

Comment:
Hmmm, with property, do you mean like setting a `attr_accessor :line_length`, and update it every time the size of the line changes?